src/components/RichTextEditor/Plugins/BoldPlugin/createBoldPlugin.ts

```diff
@@ -0,0 +1,15 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ToggleMarkPlugin, createBoldPlugin as createPlateBoldPlugin, createPluginFactory } from '@udecode/plate';
+import { BOLD_CLASSES } from './BoldMarkupElement';
+
+export const createBoldPlugin = createPluginFactory<ToggleMarkPlugin>({
```

Comment:
I think you dont need to create createPluginFactory because the Plugin already exist you can just call the createBoldPlugin and add object with props like: 
```
createBoldPlugin({
    deserializeHtml: {
            rules: [
                { validNodeName: ['SPAN'], validClassName: BOLD_CLASSES },
            ],
        },
    }
})
```
the default values should be overwritten with yours. Let me know

Reply:
fixed